payments-core/src/test/java/com/stripe/android/SourceEndToEndTest.kt

```diff
@@ -2,106 +2,16 @@ package com.stripe.android
 
 import androidx.test.core.app.ApplicationProvider
 import com.google.common.truth.Truth.assertThat
-import com.stripe.android.model.Address
 import com.stripe.android.model.CardBrand
 import com.stripe.android.model.CardParams
-import com.stripe.android.model.DateOfBirth
-import com.stripe.android.model.KlarnaSourceParams
-import com.stripe.android.model.SourceOrder
 import com.stripe.android.model.SourceParams
 import com.stripe.android.model.SourceTypeModel
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
-import kotlin.test.Ignore
 import kotlin.test.Test
 
 @RunWith(RobolectricTestRunner::class)
-@Ignore("Resume after finishing investigation in #bc-earth-embargo")
```

Comment:
Let's do this part separate from the rest of this PR. 

We're still unclear if we can remove the code.